application/cms/page_service.py

```diff
@@ -360,7 +360,6 @@ def update_measure_version(  # noqa: C901 (complexity)
             measure_version.status = status
 
         title = measure_version_form.data.pop("title").strip()
-        measure_version.title = title
```

Comment:
I think we need this here if people are ever to be able to update titles?

Reply:
Hmm, apparently not according to the tests... so how does that work if the value is popped?

Reply:
All of the posted data is injected into the measure version instance by `measure_version_form.populate_obj(measure_version)` a little further down this block.

Reply:
The code snippet this comment is on is actually a little out of date. Both these lines have been removed:

```python
        title = measure_version_form.data.pop("title").strip()
        measure_version.title = title
```

We now just do:

```python
        if measure_version.version == "1.0":
            slug = slugify(measure_version_form.title.data)

            if slug != measure_version.measure.slug and self._new_slug_invalid(measure_version, slug):
                message = f"A page with slug '{slug}' already exists under {measure_version.measure.subtopic.title}"
                raise PageExistsException(message)
            measure_version.measure.slug = slug

        # Update main fields of MeasureVersion
        measure_version_form.populate_obj(measure_version)
```